.github/workflows/add-release-reviewers.yml

```diff
@@ -21,27 +26,18 @@ jobs:
           DATA=$(gh api \
             -H "Accept: application/vnd.github+json" \
             -H "X-GitHub-Api-Version: 2022-11-28" \
-            /orgs/${{ORG}}/teams/${{TEAM_SLUG}}/members \
+            /orgs/$ORG/teams/$TEAM_SLUG/members?role=maintainer&per_page=100 \
             | jq 'reduce inputs as $i (.; . += $i)') \
-          | echo "data=$DATA" >> $GITHUB_OUTPUT \
-          | echo "members=${{ join(fromJson(DATA).*.login) }}" >> $GITHUB_OUTPUT \
-        env:
-          ORG: ${{ github.event.organization }}
-          TEAM_SLUG: x
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          echo "data=$DATA" >> $GITHUB_OUTPUT \
+          echo "members=${{ join(fromJson($DATA).*.login) }}" >> $GITHUB_OUTPUT \
```

Comment:
The ending `\` needs to be removed as well if the piping `|` gets removed

Reply:
done!